stories/DropdownLight/DropdownLight.stories.jsx

```diff
@@ -17,3 +18,22 @@ export const Disabled = Template.bind({});
 Disabled.args = {
   disabled: true,
 };
+
+export const Controlled = () => {
+  const fruits = [
+    { value: 'Lemon', label: 'Lemon' },
+    { value: 'Banana', label: 'Banana' },
+    { value: 'Strawberry', label: 'Strawberry' },
+  ];
+  const [selectedFruit, setSelectedFruit] = useState({ label: 'Banana' });
+
+  return (
+    <DropdownLight
+      label="Fruits"
+      placeholder="Select a fruit"
+      items={fruits}
+      selectedItem={selectedFruit}
```

Comment:
selected item must be an object exactly the same as an item in the list, that is, it must be a string or an object with value and label properties. The object used here has only the label